dangerzone/gui/main_window.py

```diff
@@ -55,13 +55,6 @@
 HAMBURGER_MENU_SIZE = 30
 
 
-WARNING_MESSAGE = """\
```

Comment:
Rather than removing the code, could it be better to keep it for future announces we would like to make, but for now it doesn't trigger?

Reply:
I was thinking about this as well. Generally, I prefer not to keep unused code for future-proofing, because it's pretty easy for surrounding code to change, and then you have to maintain the unused code as well.

I believe that if we do need something similar in the near future, we should be able to retrieve the code from the Git logs / PR. If such a needs happens much later on, i.e., after our UI revamp, then it will need a rewrite in any case.

Happy to hear your thoughts though, because I do find this code useful.

Reply:
Yeah. It's not a big deal anyway as it's really not a lot of code, and we currently know it's here if we need to reproduce something like it in the future.